### PR TITLE
Fix/tagged post count

### PR DESCRIPTION
**Pull request by Mehdi-Miah**

### CODE REVIEW

**Comment by Mehdi-Miah** [comment]
orchardcore.lucene is running on 1.4 latest it hasn't moved to 1.5 yet

File: Migrations.cs [hunk 1]
```diff
@@ -24,17 +28,31 @@ public async Task<int> CreateAsync()
...
+
+        public async Task<int> UpdateFrom2()
+        {
+            await _queryManager.SaveQueryAsync("ContentByTags", new LuceneQuery
+            {
+                Index = "News",
+                ReturnContentItems = true,
+                Name = "ContentByTags",
+                Template = "{\r\n  \"query\": {\r\n    \"bool\": { \r\n      \"must\": [\r\n        { \"term\": { \"NewsPost.Tags\": \"{{tagId}}\" } }\r\n      ]\r\n    }\r\n  }\r\n}"
+
```
Comment (peterkeating) [inline Migrations.cs:52]: Remove empty line.

File: Views/Tag-TermPart.liquid [hunk 2]
```diff
@@ -1,13 +1,16 @@
 ﻿{% assign termName = Model.ContentItem | display_text %}
 
+{% assign taggedContentItems = Queries["ContentByTags"] | query: tagId: Model.ContentItem.ContentItemId %}
```
Comment (peterkeating) [inline Views/Tag-TermPart.liquid:3]: Can you see if the following works, would save a unnecessary variable declaration.

```
{% assign taggedContentItemsCount = Queries["ContentByTags"] | query: tagId: Model.ContentItem.ContentItemId | size %}
```

**Review by peterkeating — CHANGES_REQUESTED** [review]

File: Etch.OrchardCore.News.csproj [hunk 3]
```diff
@@ -23,6 +23,7 @@
   <ItemGroup>
     <PackageReference Include="OrchardCore.ContentManagement" Version="1.5.0" />
     <PackageReference Include="OrchardCore.DisplayManagement" Version="1.5.0" />
+    <PackageReference Include="OrchardCore.Lucene" Version="1.4.0" />
```
Comment (peterkeating) [inline Etch.OrchardCore.News.csproj:26]: Change to target 1.5.0. We'll change it separately when we do a tag that targets 1.4.0,

**Review by peterkeating — CHANGES_REQUESTED** [review]

**Comment by Mehdi-Miah** [comment]
When did the header for NewsPost.Tags change to NewsPost.Tags.Ids 😢

File: Etch.OrchardCore.News.csproj (hunk 3 above)
Comment (Mehdi-Miah) [reply Etch.OrchardCore.News.csproj:26]: there are only pre-release versions for 1.5.0, is that ok? i just went with the stable 1.4

**Review by peterkeating — APPROVED** [review]

**Pull request merged**